### PR TITLE
Position marker SVG graphics

### PR DESCRIPTION
**Pull request by Pikkuninja**

### CODE REVIEW

File: app/js/controllers/mapController.js [hunk 1]
```diff
@@ -73,15 +73,21 @@ angular.module('cityInfo.controllers').controller('MapController', ['$scope', 'M
...
+    else {
+      latLng.lat = data.latitude;
+      latLng.lng = data.longitude;
+    }
 
     $scope.latestLocation = latLng;
     if ($scope.map !== null) {
       if (userPosMarker === null) {
-        userPosMarker = $scope.map.createMarker(latLng, 'img/userPositionMarker.png');
+        userPosMarker = $scope.map.createMarker(latLng,'img/userpositionmarker.svg');
```
Comment (heinoapt) [inline app/js/controllers/mapController.js]: Space between the parameters would clean this up a bit!

Comment (Pikkuninja) [reply app/js/controllers/mapController.js]: whoops

**Review by heinoapt — APPROVED** [review]
Looks good!

**Comment by heinoapt** [comment]
Reviewed and merged.

**Pull request merged**